webapp/src/components/HomePage/Sidebar/ExpandedSidebar/WalletParcels/ParcelTable/ParcelTable.js

```diff
@@ -26,9 +26,7 @@ class ParcelTable extends React.PureComponent {
         </div>
       )
     } else {
-      return (
-        <div className="table-row-empty">You have no available land yet</div>
-      )
+      return <div className="table-row-empty">{"You don't own any land."}</div>
```

Comment:
Why `{}` here? ( for `{"You don't own any land."}` )

Reply:
it was cos of the single quote :P but yea I can alwas `&apos;`